02 - XML.md

```diff
@@ -226,6 +228,13 @@ La seguente figura mostra i gradi di relazione che sussistono tra gli standard c
 
 
 
+Analisi:
+
+Si hanno i tags `<bibliography>` simile all HTML ma la differenza è che in questo caso si può usare qualsiasi tags che si vuole, tuttavia deve essere presente nel DTD che è una sorta di dizionario.
```

Comment:
> Si hanno i tags `<bibliography>` simile all HTML ma la differenza è che in questo caso si può usare qualsiasi tags che si vuole, tuttavia deve essere presente nel DTD che è una sorta di dizionario.

Ci sono i tags `<bibliography>` che sono simili a quelli già visti per HTM, la differenza è che in questo caso si può usare qualsiasi tag personalizzato, tuttavia deve essere presente nel DTD che rappresenta il modello del documento.